src/__vendor/a9-apstag.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Load Amazon A9 library as {@link https://ams.amazon.com/webpublisher/uam/docs/web-integration-documentation/integration-guide/javascript-guide/display.html described here}
+ */
+export const a9Apstag = () => {
+	!(function(a9, a, p, s, t, A, g) {
```

Comment:
Don't think the `!` is doing anything useful here?:
```suggestion
	(function(a9, a, p, s, t, A, g) {
```

Reply:
Nice, I'm pretty sure we shouldn't ever be returning (as we're not here) and using the return value of these stubs regardless, so I agree. I wonder why this and the `;` in your other comment were there originally 🤔

Reply:
I think the `;` was an old best practise to do with minifying/concatenating scripts, just in case the script before yours forgot it's final `;` it wouldn't break everything. Possibly a similar story for the `!` 🤔

Reply:
I think the `!`  was an old byte-saving IIFE trick so you could do `!function(){}()` without wrapping the function and it would invoke. 

But given we have `(function(){})()` I think it's not needed regardless.